cromshell

```diff
@@ -711,21 +711,23 @@ function slim-metadata()
 # Get the status of the given job and how many times it was run
 function execution-status-count() 
 {
-  assertCanCommunicateWithServer $2
+  assertCanCommunicateWithServer "$2"
```

Comment:
I know it's somewhat bad form, but can you remove the quotes from the URL for the server here (to make it consistent with the rest of the script)?